index.html

```diff
@@ -18,7 +18,7 @@ <h1 class="tagline col-md-5">
             <i class="fa fa-plus"></i>
             <img src="{{ site.baseurl }}/assets/images/foreman.png" alt="Foreman"/>
           </p>
-          <a href="{{ site.baseurl }}/docs/2.0/installation/index.html" class="get-started">
+          <a href="{{ site.baseurl }}/docs/{{ site.latest }}/installation/index.html" class="get-started">
```

Comment:
The other benefit here is from unversioned parts of the site can link to "/docs/{{ site.latest }}/whatever" now so you don't link users to nightly, but to latest stable.  The idea came from Foreman's site, which does something similar.